CONTRIBUTING.md

```diff
@@ -15,6 +15,13 @@ This also applies to the code snippets in the markdown files.
 
 * Please wrap the code at 72 characters.
 
+## Comments
```

Comment:
Hi @jieyu,

I think the style of comments in Makefiles vs Source files vs etc. can be different. As long as the intent is understood and a basic style is adhered (line length, wrapping, etc), I would relax a little.